src/main/java/com/codingnomads/betty/data/api/TwitterSourceMinerRepository.java

```diff
@@ -37,4 +37,37 @@ public TwitterSourceMinerRepository(TwitterConfig twitterConfigurer) {
 
         return tweets;
     }
+
+    public List<Status> searchTweetFromAccounts(String keyword, String startDate) {
+        Queue<String> queries = new LinkedList<>();
+        queries.add("Chelsea -Arsenal from:ChelseaFC");
+        queries.add("Chelsea -Arsenal from:BBCSport");
+
+        Twitter twitter = twitterConfigurer.getTwitter();
+        int index = 1;
+        List<Status> tweets = new ArrayList<>();
+
+        while (!queries.isEmpty()) {
+            Query query = new Query(queries.poll());
+            query.setLang("en");
+            query.setSince(startDate);
+
+            QueryResult queryResult = null;
+            try {
+                queryResult = twitter.search(query);
+            } catch (TwitterException te) {
+                throw new TwitterSearchFailedException("Tweet Search Failed!", te);
+            }
+
+            tweets = queryResult.getTweets();
+
+            for (Status tweet : tweets) {
+                System.out.println(index++ + "--------------------------");
```

Comment:
Is there a reason why you kept these souts here? Maybe we can create a custom exception here to check these fields are populated instead of 'souting' them? :)

Reply:
Can I keep these for now? These were quite helpful for testing the twitter queries, and I found myself writing these back in when building another feature earlier too. I will delete them in a future PR when everything else is integrated :)